File: json/json.rb
require 'json'

file = "input.json"
json_str = ""
File.open(file, "r") do |rfh|
    json_str = rfh.read()
end
djson = JSON.parse(json_str)
for item in djson["items"]
    print(item["title"]+"\n")
end

File: examples/json/json.js
const fs = require('fs');
let file = "./input.json";
fs.readFile(file, (err, data) => {
if (err) throw err;
    const obj = JSON.parse(data.toString());
    for(const item of obj["items"]) {
        console.log(item["title"]);
    }
});
